src/catalog/src/durable/transaction.rs

```diff
@@ -766,6 +766,22 @@ impl<'a> Transaction<'a> {
         }
     }
 
+    pub fn remove_databases(&mut self, ids: &BTreeSet<DatabaseId>) -> Result<(), CatalogError> {
+        let to_remove = ids
+            .iter()
+            .map(|id| (DatabaseKey { id: *id }, None))
+            .collect();
+        let prev = self.databases.set_many(to_remove, self.op_id)?;
+
+        for (database_key, prev) in prev {
+            if prev.is_none() {
+                return Err(SqlCatalogError::UnknownDatabase(database_key.id.to_string()).into());
+            }
+        }
```

Comment:
The error message format and doc-comments for this and other added methods to this file are slightly different from the existing plural remove methods (for example `remove_items` https://github.com/MaterializeInc/materialize/blob/9ebe2787cb794726bc2ddfa7204f49c82010c2e0/src/catalog/src/durable/transaction.rs#L866-L872). It's not a huge deal, but it would be nice to keep these in sync.

Reply:
Yeah definitely a good idea to keep these in-sync, I updated them